packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -2,6 +2,7 @@
 
   :host { 
     display: inline-block;
+    width: 100%;
```

Comment:
Instead of adding `width: 100%`, you can just replace `inline-block` with `block`.